lib/tasks/before_assets_precompile.rake

```diff
@@ -1,3 +1,4 @@
+desc nil
```

Comment:
Was this added automatically or is it a rubocop thing?

Reply:
it's just to suppress rubocop. Not pretty, I know...

Reply:
Could we add a little description "Before assets precompile" - redundant but maybe nicer than `nil`?